core.js

```diff
@@ -2782,13 +2972,42 @@ class HTMLVideoElement extends HTMLMediaElement {
   }
 
   get width() {
-    return 0;
+    return this.video ? this.video.width : 0;
   }
   set width(width) {}
   get height() {
-    return 0;
+    return this.video ? this.video.height : 0;
   }
   set height(height) {}
+
+  get autoplay() {
+    return this.getAttribute('autoplay');
+  }
+  set autoplay(autoplay) {
+    this.setAttribute('autoplay', autoplay);
+  }
+
+  getBoundingClientRect() {
+    return new DOMRect(0, 0, this.width, this.height);
+  }
+
+  get data() {
+    return this.video ? this.video.data : null;
+  }
+  set data(data) {}
+
+  play() {
+    if (this.video)
```

Comment:
Curlification? That's kind of the standard formatting scheme in the rest of the code base.